RGO.App.Tests/Controllers/HRIS/EmployeeRoleManagerControllerUnitTests.cs

```diff
@@ -105,7 +105,7 @@ public async Task AddRoleExistingRoleReturnsCreatedAtAction()
         employeeServiceMock.Setup(x => x.GetEmployee(email)).ReturnsAsync(CreateEmployee(email));
 
         roleServiceMock.Setup(x => x.CheckRole(role)).ReturnsAsync(true);
-        roleServiceMock.Setup(x => x.GetRole(role)).ReturnsAsync(new RoleDto(1, "Super Admin"));
+        roleServiceMock.Setup(x => x.GetRole(role)).ReturnsAsync(new RoleDto{ Id = 0, Description = "Super Admin" });
```

Comment:
Just double check that you initialize the correct values everywhere
(Id was 1 you initialized it as 0)